server/middlewares/checkAuth.js

```diff
@@ -0,0 +1,31 @@
+import jwt from 'jsonwebtoken';
+import config from '../config/config';
+import { badHttpResponse } from '../utilities/httpResponse';
+
+const env = process.env.NODE_ENV || 'development';
```

Comment:
seen this duplicated in a number of places. Can we make it a helper? `getCurrentEnv()`

Reply:
Okay